Chinese/Yaoming.md

```diff
@@ -0,0 +1 @@
+转眼姚明都已经退役好几年了，科比也要退役了，老一代基本剩不下什么人了，伊戈达拉都要退役了，真的是老了。
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.